multiqc/modules/samtools/flagstat.py

```diff
@@ -37,13 +37,23 @@ def parse_samtools_flagstats(self):
 
             # General Stats Table
             flagstats_headers = dict()
+            flagstats_headers['flagstat_total'] = {
+                'title': '{} Total Reads'.format(config.read_count_prefix),
+                'description': 'Total Mapped in the bam file ({})'.format(config.read_count_desc),
```

Comment:
I don't think that they necessarily have to be mapped?

```suggestion
                'description': 'Total reads in the bam file ({})'.format(config.read_count_desc),
```